pkg/controlplane/instance_test.go

```diff
@@ -399,68 +399,6 @@ func TestStorageVersionHashes(t *testing.T) {
 	}
 }
 
-func TestStorageVersionHashEqualities(t *testing.T) {
-	apiserver, etcdserver, _, assert := newInstance(t)
-	defer etcdserver.Terminate(t)
-
-	server := httptest.NewServer(apiserver.GenericAPIServer.Handler.GoRestfulContainer.ServeMux)
-
-	// Test 1: extensions/v1beta1/ingresses and apps/v1/ingresses have
```

Comment:
interestingly, these compared ok, likely because they both 404'd at the same time.

Reply:
that's funny